css/style.css

```diff
@@ -61,75 +58,83 @@ div {
 
 .last-name {
   height: 0;
-  margin-top: 5px;
-  margin-bottom: 35px;
-  margin-left: 19px;
+  margin-top: 10px;
+  margin-bottom: 25px;
   font-family: "Roboto", sans-serif;
   font-weight: 500;
   font-size: 30px;
   font-style: normal;
   color: white;
+  text-transform: uppercase;
 }
+
 .job-tittle {
   height: 0;
-  margin-left: 19px;
   font-family: "Roboto", sans-serif;
   font-weight: 400;
   font-size: 25px;
   font-style: normal;
+  text-transform: uppercase;
   color: #6487A1;
 }
 
-.box-4-contact-info {
+.box-contact-info {
+  padding-left: 20px;
   display: flex;
   justify-content: center;
   flex-direction: column;
   border-bottom-right-radius: 10px;
-  width: 383px;
-  height: 190px;
+  width: 363px;
+  height: 188px;
   color: white;
   background-color:  #2B3944;
+  border-top: 2px solid white;
 }
 
-.map-pin {
-  display: flex;
-  flex-wrap: wrap;
-  order: -1;
-  margin-left:-35px ;
-  margin-right: 10px;
+.contact-item::after {
+  content: "";
+  display: block;
+  width: calc(100% - 30px);
+  height: 1px;
+  background-color: var(--semi-transparent-white);
+  position: absolute;
+  bottom: 3px;
+  left: 38px;
 }
-.address {
-  display: flex;
-  justify-content: flex-start;
-  align-content: flex-end;
-  border-bottom: 1px solid rgb(255,255,255, 0.4);
-  margin-left: 55px;
+
+.contact-info {
+  width: 100%;
+  max-width: 400px;
+  padding: 20px;
 }
-.mail-logo {
+
+.contact-item {
+ background: #2B3944;
+  height: 30px;
   display: flex;
-  flex-wrap: wrap;
-  order: -1;
-  margin-left:-35px ;
-  margin-right: 10px;
+  align-items: center;
+  margin-bottom: 15px;
+  position: relative;
+  padding-bottom: 5px;
 }
-.email {
-  display: flex;
-  justify-content: flex-start;
-  align-content: flex-end;
-  border-bottom: 1px solid rgb(255, 255, 255, 0.4);
-  margin-left: 55px;
+
+.icon {
+  margin-right: 15px;
+  width: 25px;
+  height: auto;
 }
-.phone-logo {
-  display: flex;
-  flex-wrap: wrap;
-  order: -1;
-  margin-left:-35px ;
-  margin-right: 10px;
+
+:root {
+  --semi-transparent-white: rgba(255, 255, 255, 0.4); /* Define the CSS variable */
```

Comment:
We haven't used the CSS variables yet. It is not a bad habit to use those. Just make sure that you understand how it works. Apart from that you don't need to add comments in the code which explains the syntax.

It looks like copied from chat GPT

Reply:
I can't find your original comments you've made on the assignment, but as far as I remember you requested to implement VAR and to google it how to use it.